components/main/index.tsx

```diff
@@ -0,0 +1,70 @@
+import styled from 'styled-components';
+
+import Rank from '@components/rank';
+import Sidebar from '@components/sidebar';
+
+import { device } from '@utils';
+import type { Data } from '@utils';
+
+const MainWrapper = styled.main`
+	margin-top: 1.375rem;
+	display: grid;
+	grid-template-rows: 100px 1fr;
+	gap: 1rem;
+
+	@media ${device.tablet} {
+		grid-template-columns: 70% 1fr;
+		grid-template-rows: auto;
+	}
+`;
+
+const RankWrapper = styled.div`
```

Comment:
html nit: You could consider `section` element instead of `div` for `RankWrapper` and `SidebarWrapper`

I'm impressed with the overall the HTML though (:

Reply:
https://github.com/ardriveapp/inferno-leaderboard/pull/2/commits/8bf56c0a9a8e4a51c59860a472163fddc32061c9